### ISSUE TITLE
508-defect-2 [COGNITION]: App Directory - Accessible labels for "Find app" should describe which app

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16441]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As an assistive technology user, I want to understand what app I'll be looking for when I click the "Find app" link.

## Details

We have several apps on the page that all have a call to action link that says "Find app". Sighted users can make a correlation between the heading and the call to action, but we could add more context for assistive technology with an `aria-label` that uses the visual label plus descriptive text. Screenshot and code snippet below.

## Acceptance Criteria

- [ ] aria-labels added for each "Find app" link
- [ ] No axe browser violations

## Solution (if known)

```diff
<a
+ aria-label="Find app Apple Health"
  class="usa-button usa-button-secondary vads-u-width--auto"
  href="https://www.apple.com/ios/health/"
  rel="noopener noreferrer"
  target="_blank"
>Find app</a>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-11-19 at 3 51 29 PM](https://user-images.githubusercontent.com/934879/99728527-326ab800-2a7f-11eb-9f05-1a3da221591f.png)

**Comment by LeKeve** [comment]
Ready to test in production.

**Comment by 1Copenut** [comment]
LGTM. Moving to close.
